app/src/ActionModal/components/ModalVoteSubstrate.vue

```diff
@@ -0,0 +1,428 @@
+<template>
+  <!-- <SessionFrame :icon="`thumbs_up_down`"> -->
+  <ActionModal
+    id="modal-vote-substrate"
+    ref="actionModal"
+    :validate="validateForm"
+    title="Vote"
+    class="modal-vote"
+    submission-error-prefix="Voting failed"
+    :transaction-type="messageType.VOTE"
+    :transaction-data="transactionData"
+    :notify-message="notifyMessage"
+    feature-flag="vote"
+    @close="clear"
+    @txIncluded="onSuccess"
+  >
+    <div class="action-modal-group vote-options">
+      <h3>Select your vote</h3>
+      <div class="flex-row">
+        <TmBtn
+          id="vote-yes"
+          :class="{ active: vote === `Yes` }"
+          :disabled="lastVoteOption === `Yes`"
+          color="secondary"
+          value="Yes"
+          size="md"
+          @click.native="vote = 'Yes'"
+        />
+        <TmBtn
+          id="vote-no"
+          :class="{ active: vote === `No` }"
+          :disabled="lastVoteOption === `No`"
+          color="secondary"
+          value="No"
+          size="md"
+          @click.native="vote = 'No'"
+        />
+      </div>
+    </div>
+    <div class="line"></div>
+    <div class="locking-area">
+      <p>
+        Available to vote <span>{{ stakingDenomBalance.available }}</span>
+      </p>
+      <div class="locked-balance-container">
+        <TmFormGroup
+          :error="$v.lockedBalance.$error && $v.lockedBalance.$invalid"
+          class="action-modal-group"
+          field-id="lockedBalance"
+        >
+          <TmField
+            id="lockedBalance"
```

Comment:
why use an id here?

Reply:
removing

Reply:
at the end I need this to text-align the balance within the input